src/app/board/page.js

```diff
@@ -1,5 +1,27 @@
+import Board from "@/components/board/Board";
+import { BOARD } from "@/data/boardMembers";
+import Title from "@/components/Title";
+
 const Page = () => {
-  return <div>Board</div>;
+  return (
+    <div>
+      <div>
+        <Title text="Get To Know Us" />
+        <div className="grid lg:grid-cols-3 md:grid-cols-2 sm:grid-cols-1 gap-y-16 gap-x-20 mx-16 my-16">
```

Comment:
```suggestion
        <div className="grid lg:grid-cols-3 md:grid-cols-2 sm:grid-cols-1 gap-y-16 gap-x-20">
```
don't use margining to position elements.